src/extension/inpage-script/index.js

```diff
@@ -121,6 +121,27 @@ if (document) {
     },
     { capture: true }
   );
+  // Listen for webln events from the extension
+  // emit events to the websites
+  window.addEventListener("message", (event) => {
+    if (event.source === window && event.data.action === "accountChanged") {
+      if (
+        (window.webln && window.webln.enabled) ||
```

Comment:
I think these checks should be moved into the eventEmitter, it would be safer to check the scope and check if the window object (nostr or webln) exists in the same place

Reply:
resolved